gemet/thesaurus/tests/test_import.py

```diff
@@ -0,0 +1,35 @@
+from django.core.files import File
+from django.test import TestCase, Client
+
+from .factories import VersionFactory
+from gemet.thesaurus.models import Concept, Import
+
+
+class ConceptImportView(TestCase):
+
+    # Create namespaces, languages, property types, and users
+    fixtures = ['data.json']
+
+    def setUp(self):
+        VersionFactory(identifier='')
+        self.client = Client()
+
+    def test_import(self):
```

Comment:
This test is failing in the Travis suite: https://travis-ci.org/github/eea/gemet/builds/753175071